src/core/rtkQuery/splitV3Api/seriesApi.ts

```diff
@@ -200,6 +200,15 @@ const seriesApi = splitV3Api.injectEndpoints({
       }),
     }),
 
+    changeSeriesImage: build.mutation<ResponseType, { seriesId: string, imageType: string, params: object }>({
```

Comment:
Suggestion; use `image: Pick<ImageType, 'ID' | 'Source' |'Type">` instead of `imageType: string, params: object`, then map the properties of `image` where they are needed. This will make it so we can just give the image object to the mutation method when using the mutation.

Reply:
Thanks for the tip!